packages/core/src/common/utils.ts

```diff
@@ -94,3 +94,10 @@ export function throttleEvent(target: EventTarget, eventName: string, newEventNa
     target.addEventListener(eventName, func);
     return func;
 };
+
+/**
+ * Returns true if at least one item in the array satifies the predicate.
+ */
+export function some(a: any[] = [], predicate: (item: any) => boolean) {
```

Comment:
Isn't this a thing that already exists: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?v=control ?